libs/api/domains/auth-admin/src/lib/shared/services/multi-environment.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Inject, Injectable } from '@nestjs/common'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import type { Logger } from '@island.is/logging'
+import {
+  AdminApi,
+  AdminDevApi,
+  AdminProdApi,
+  AdminStagingApi,
+} from '@island.is/clients/auth/admin-api'
+import { Auth, AuthMiddleware } from '@island.is/auth-nest-tools'
+
+@Injectable()
+export class MultiEnvironmentService {
```

Comment:
We should have this as an `abstract` class
```suggestion
export abstract class MultiEnvironmentService {
```